### ISSUE TITLE
Sunstone: can't display VM information with SCHED_ACTION

### ISSUE THREAD

**Issue by baby-gnu** [OpenNebula/one#6128]
**Description**

When a VM has some scheduled actions, it can't be displayed in the Sunstone interface (no Fireedge).

**To Reproduce**

1. Create a VM
2. Add a schedule action: `onevm terminate 3 --schedule "06/07"`
3. Open Sunstone
4. Display the VM: `https://one.example.net/#vms-tab/3`

**Expected behavior**

The VM information are displayed.

**Details**
 - Affected Component: Sunstone
 - Hypervisor: KVM
 - Version: 6.6

**Additional context**

<details><summary>firefox render a loading progress bar indefinitely</summary>

![sunstone-vm-with-sched_action-error](https://user-images.githubusercontent.com/1233212/218443595-f68a7352-3da0-43ab-a97c-77672eaaf9a3.png)

</details>

<details><summary>firefox consol error log</summary>

```
Uncaught ReferenceError: startTime is not defined
    leasesClock datatable-common.js:162
    leasesClock datatable-common.js:151
    _elementArray datatable-common.js:250
    _elementArray datatable.js:123
    _updateElement tab-datatable.js:703
    callback actions.js:44
    success action.js:319
    jQuery 6
    show action.js:312
    show vm.js:466
    _runAction sunstone.js:984
    _routerShowElement sunstone.js:817
    routeWithID sunstone.js:1347
    resolve tips.js:37
    r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
    setTimeout handler*r tips.js:37
datatable-common.js:162
    leasesClock datatable-common.js:162
    some self-hosted:137
    leasesClock datatable-common.js:151
    _elementArray datatable-common.js:250
    _elementArray datatable.js:123
    _updateElement tab-datatable.js:703
    callback actions.js:44
    success action.js:319
    jQuery 6
    show action.js:312
    show vm.js:466
    _runAction sunstone.js:984
    _routerShowElement sunstone.js:817
    routeWithID sunstone.js:1347
    routeWithID self-hosted:1362
    resolve tips.js:37
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
    (Asynchrone : setTimeout handler)
    r tips.js:37
```
</details>

## Progress Status
- [x] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by baby-gnu** [comment]
I have the same issue with chromium `110.0.5481.77-2` from Debian unstable.

**Comment by wikti** [comment]
I think it might be regression. Sunstone properly displays information in version 6.0, when actions are added/modified by xml api.

**Comment by jloboescalona2** [comment]
Is solved in the following PR: https://github.com/OpenNebula/one-ee/pull/2512

**Comment by tinova** [comment]
This should be solved in 6.8 Beta1, please let us know if this is not the case and we'll reopen it
